packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -273,6 +274,84 @@ export abstract class RestApiBase extends Resource implements IRestApi {
     return this._domainName;
   }
 
+  /**
+   * Return the given named metric for this API
+   */
+  public static metricAll(metricName: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return new cloudwatch.Metric({
+      namespace: 'AWS/ApiGateway',
+      metricName,
+      ...props,
+    });
+  }
+
+  /**
+   * Metric for the number of client-side errors captured in a given period.
+   *
+   * @default sum over 5 minutes
+   */
+  public static metricAll4XXError(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metricAll('4XXError', { statistic: 'Sum', ...props });
+  }
+
+  /**
+   * Metric for the number of server-side errors captured in a given period.
+   *
+   * @default sum over 5 minutes
+   */
+  public static metricAll5XXError(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metricAll('5XXError', { statistic: 'Sum', ...props });
+  }
+
+  /**
+   * Metric for the number of requests served from the API cache in a given period.
+   *
+   * @default sum over 5 minutes
+   */
+  public static metricAllCacheHitCount(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metricAll('CacheHitCount', { statistic: 'Sum', ...props });
+  }
+
+  /**
+   * Metric for the number of requests served from the backend in a given period,
+   * when API caching is enabled.
+   *
+   * @default sum over 5 minutes
+   */
+  public static metricAllCacheMissCount(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metricAll('CacheMissCount', { statistic: 'Sum', ...props });
+  }
+
+  /**
+   * Metric for the total number API requests in a given period.
+   *
+   * @default SampleCount over 5 minutes
+   */
+  public static metricAllCount(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metricAll('Count', { statistic: 'SampleCount', ...props });
+  }
+
+  /**
+   * Metric for the time between when API Gateway relays a request to the backend
+   * and when it receives a response from the backend.
+   *
+   * @default - no statistic
```

Comment:
What does it mean by 'no statistic'?

Reply:
No statistic means there is no default statistic that is configured for this metric. There is no aggregation done by couldwatch for this metric (https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#Statistic)